socorro/external/crashstorage_base.py

```diff
@@ -459,7 +459,7 @@ def __setitem__(self, index, value):
         self.exceptions.__setitem__(index, value)
 
     def __str__(self):
-        return '%s %s' % (self.message,
+        return '%s %s' % (self.args[0],
```

Comment:
Because the "message" argument as an attribute is deprecated, the suggested way to use the message is `args[0]`, so I switched to that here and elsewhere.